src/core/optimization_container.jl

```diff
@@ -303,14 +307,39 @@ function build_impl!(container::OptimizationContainer, template, sys::PSY.System
         #  TODO: Add dual variable container for services
     end
     for device_model in values(template.devices)
-        @debug "Building $(get_component_type(device_model)) with $(get_formulation(device_model)) formulation"
+        @debug "Building Arguments for $(get_component_type(device_model)) with $(get_formulation(device_model)) formulation"
```

Comment:
@debug "Building Arguments for $(get_component_type(device_model)) with $(get_formulation(device_model)) formulation" _group = :ConstructGroup